core/pom.xml

```diff
@@ -616,6 +616,26 @@
         <script.extension>.sh</script.extension>
       </properties>
     </profile>
+    <profile>
+      <id>opentelemetry-reporter</id>
+      <dependencies>
+        <dependency>
+          <groupId>io.opentelemetry</groupId>
+          <artifactId>opentelemetry-sdk</artifactId>
+          <version>1.40.0</version>
```

Comment:
Let's use the latest one.
- https://mvnrepository.com/artifact/io.opentelemetry/opentelemetry-api/1.41.0

Reply:
Updated